frontend/app/views/support/Dates.scala

```diff
@@ -59,18 +59,31 @@ object Dates {
 
   case class Range(start: String, end: String)
 
-  def dateRange(start: DateTime, end: DateTime): Range = {
-    def dateToks(dt: DateTime) = dt.toString("EEEE d MMMMM YYYY").split(" ").reverse
+  private def buildDateRange(start: DateTime, end: DateTime, fmt: String): Range = {
```

Comment:
The eventual behaviour was clear (and very nice), just not what the particular part of code controlled so that helps a bit.